sdcm/utils/health_checker.py

```diff
@@ -33,26 +33,17 @@
 HealthEventsGenerator = Generator[ClusterHealthValidatorEvent, None, None]
 
 
-def check_nodes_status(nodes_status: dict, current_node, removed_nodes_list=None) -> HealthEventsGenerator:
+def check_nodes_status(nodes_status: dict, current_node) -> HealthEventsGenerator:
     node_type = 'target' if current_node.running_nemesis else 'regular'
     if not nodes_status:
         LOGGER.warning("Node status info is not available. Search for the warning above")
         return
-
     LOGGER.info("Status for %s node %s", node_type, current_node.name)
-
-    if removed_nodes_list is None:
-        removed_nodes_list = ()
-
     for node_ip, node_properties in nodes_status.items():
         if node_properties['status'] != "UN":
             is_target = current_node.print_node_running_nemesis(node_ip)
-
-            # FIXME: #2383 must be reverted once scylladb/scylla-enterprise#1419 will be fixed.
-            LOGGER.debug("REMOVED NODES LIST = %s", removed_nodes_list)
-
             yield ClusterHealthValidatorEvent.NodeStatus(
-                severity=Severity.ERROR if node_ip in removed_nodes_list else Severity.CRITICAL,
```

Comment:
Maybe it will be easier to debug this or similar issue if we keep that removed_nodes_list here  as a INFO log message if that  list isn't empty.